client/tests/unit/views/CreateCopyrightApplication.spec.js

```diff
@@ -84,5 +86,5 @@ describe('Create Copyright Application', () => {
 
     expect(submittedApplication.correspondencePhoneNumber).to.equal('(410) 555 1234')
     expect(submittedApplication.correspondenceEmail).to.equal('tooKool@4skool.io')
-  })
+  }).timeout(5000)
```

Comment:
What's the timeout for?

Reply:
My computer was having an off-day. It was taking longer than the default 2-second timeout to run the test. Probably because of all the assertions. I'll remove it and see how it goes.

Reply:
moved the timeout to package.json since CI ran into this issue